microprofile/microprofile-jwt-1.1/src/test/java/org/wildfly/swarm/ts/microprofile/jwt/v11/MPConfigPublickeyJwt11Test.java

```diff
@@ -0,0 +1,39 @@
+package org.wildfly.swarm.ts.microprofile.jwt.v11;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.ClassLoaderAsset;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+public class MPConfigPublickeyJwt11Test extends AbstractMPConfigJwt11Test {
+
+    @Deployment
+    public static Archive<?> deployment() {
+        String mpConfigProperties = "mp.jwt.verify.publickey=MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAuvtFLWirJ/eSn9ZNdBuf" +
+                "gnynStau9zHbOj/oDbLfISfMKcG+0Pli8405Ax6y8wWdUElSS/t6sHWe/7sz/yme" +
+                "EhGshbhpJA5E2xzm/N2d0xkzTNeUDhShABIFbmEAK6cRipZaBiNw7kHiwnBe/c4C" +
+                "RekD4erZqtLu20QjF2roF8Xo1xG9klWQdxXcnAllWJXyCITlBq/5IE+X9UAhQkYW" +
+                "BNMnYNHXZk2uHjeXqL7RSeZ+CdMjRIiHrJj6+IHAf7V4Ax4dpIaGBo5yAG0ffit1" +
+                "7pmoeha3ap5UXWwLwmH/oAjzhN3MRHnaqgfVTVxzNUiX2b5DtVhworfEwqltnGrM" +
+                "ZO2EXAxDUUUb7o0qsit7PJ7KVVmjHprMjGvoTkGO2x6FKvW+4kcP/zcmUk1HBgli" +
+                "MDBndnAgyQahQjn1cKaRECIZtZWKtRqm+crzC4ioP8kU35XVMk0yeJlymy+oR3xj" +
+                "E23QOFK+4/JaC8rh7UMGirrs+MqHDPAyXTa40Rx6bRUZujlghtwjWaURCjB2Di/z" +
+                "0JviMZ61puNJZzWEvD7qRIJEkk1CYjx15CseSMZEFHBDUZyEczyuBb11eVYV6QVj" +
+                "q22HFEf8lDSMDadXH//4uTgWryCqeOWmGdXhrUf5QNzT0xN/eivBY+GJLO5GvAeT" +
+                "cSshnfu7EcJKVPKUqkC8BlkCAwEAAQ==\n" +
+                "mp.jwt.verify.issuer=" + ISSUER;
```

Comment:
This should be a file in `src/test/resources` and a `ClassLoaderAsset` instead of a `String`/`StringAsset`.